ntp-proto/src/nts_record.rs

```diff
@@ -2175,7 +2285,12 @@ mod test {
         assert_eq!(result.port, 123);
     }
 
-    fn client_server_pair() -> (KeyExchangeClient, KeyExchangeServer) {
```

Comment:
Note: this test code was modified so it can also create an authorised client. We re-use the test-certificate of the server for this.